rwengine/src/script/ScriptMachine.hpp

```diff
@@ -153,7 +164,8 @@ class ScriptMachine {
     SCMFile* file;
     ScriptModule* module;
     GameState* state;
-
+    bool debugFlag;
```

Comment:
Are we sure this should be in `ScriptMachine`? Do we know that the opcodes don't modify a game-wide debug mode? Is there any documentation for this?

Reply:
Read above.